db/test.js

```diff
@@ -0,0 +1,206 @@
+const tableColumns = {
+  //slice 3->35
+  name: "String",
+  //select from array index: 0-100
+  price: "Number",
+  //slice 3->70
+  bulletOne: "String",
+  //slice 3->70
+  bulletTwo: "String",
+  //slice 3->70
+  bulletThree: "String",
+  //slice 3->35
+  sellerName: "String",
+  //slice 3->255
+  description: "String",
+  productID: "Number"
+};
+
+const nameString =
```

Comment:
I'd recommend putting all dummy data in a seperate file, even if it is just a test.